app/styles/cards.module.css

```diff
@@ -184,8 +184,17 @@
 
   .modal {
     height: 85vh;
-    overflow: scroll;
+    overflow-y: scroll;
     width: 350px;
+    scrollbar-width: none;
+  }
+
+  .modal::-webkit-scrollbar {
+    display: none;
+  }
+
+  .modal::-webkit-scrollbar-thumb {
+    display: none;
```

Comment:
Does this works on other browser also

Reply:
Yes for mozilla scrollbar-width property is set to none